azurerm/resource_arm_service_fabric_cluster_test.go

```diff
@@ -432,6 +467,34 @@ resource "azurerm_service_fabric_cluster" "test" {
 `, rInt, location, rInt, count)
 }
 
+func testAccAzureRMServiceFabricCluster_manualClusterCodeVersion(rInt int, location, clusterCodeVersion string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name = "acctestRG-%[1]d"
```

Comment:
Very minor but could we align these assignments?

Reply:
sorry copy and paste job